app/code/Magento/Newsletter/view/adminhtml/templates/preview/iframeswitcher.phtml

```diff
@@ -21,7 +21,7 @@
             frameborder="0"
             title="<?= $block->escapeHtmlAttr(__('Preview')) ?>"
             width="100%"
-            sandbox="allow-forms allow-pointer-lock"
+            sandbox="allow-same-origin allow-scripts allow-forms allow-pointer-lock"
```

Comment:
Hi @edenduong. Do we really need it?

Reply:
Hi @VladimirZaets : we need it because we need access the attribute of iframe by javascript. Now some code execute Js to get the height of Iframe html. Add this code will allow for that without console error.